### ISSUE TITLE
[Bug] Generic Weather: Enable integration in weather app, then sync

### ISSUE THREAD

**Issue by NicholasFlamy** [pacjo/SmartspacerPlugins#20]
![Screenshot_20241215_120203_Lawnchair](https://github.com/user-attachments/assets/d80c943b-0509-4e84-ae72-1e55f98e3c90)
![Screenshot_20241215_120251_Breezy Weather](https://github.com/user-attachments/assets/a6e067bb-05ba-4f55-8b30-ccbe6c9aa439)

Non-rooted Samsung Galaxy S22 OneUI 6.1
Maybe I screwed with it, stuff like ADB uninstalling the Google Play Store.

**Comment by pacjo** [comment]
Huh, nothing's changed recently, so it should just work. Did you refresh the weather after turning the option on?

Also are you running the latest breezy weather? (though that shouldn't matter)

**Comment by NicholasFlamy** [comment]
Strange thing is, it worked for a little while a month or two ago when I first set it up, but stopped working at some point not long after I set it up and hasn't worked ever since.

**Comment by NicholasFlamy** [comment]
> Huh, nothing's changed recently, so it should just work. Did you refresh the weather after turning the option on?

Yes

> Also are you running the latest breezy weather? (though that shouldn't matter)

Yes

**Comment by NicholasFlamy** [comment]
I just checked and you release v1.5 but I'm on v1.4. Maybe I should try updating the plugin? Not sure how to do that through Smartspacer but I could manually update it.

**Comment by NicholasFlamy** [comment]
Nevermind I was on v1.5, your info page still says v1.4

**Comment by pacjo** [comment]
> Nevermind I was on v1.5, your info page still says v1.4

Oh. Yeah, that's a bug. I'll fix it when I'll have some free time.

You can try resetting app data ore reinstalling the plugin completely. I'm not sure what might have caused it to break.

**Comment by NicholasFlamy** [comment]
> > Nevermind I was on v1.5, your info page still says v1.4
> 
> Oh. Yeah, that's a bug. I'll fix it when I'll have some free time.
> 
> You can try resetting app data ore reinstalling the plugin completely. I'm not sure what might have caused it to break.

Yeah, I tried that, even downgraded to v1.4 and back to v1.5, nothing.
May have to reset Smartspacer.

**Comment by NicholasFlamy** [comment]
Holy sh*t it just started working. Completely out of the blue. Maybe the Smartspacer update or maybe Brezzy Weather update (I use Obtainium for automatic updates).

**Comment by pacjo** [comment]
Given that it works for you now (and I haven't got any additional reports for it) I'm going to close this.

A bit of a pity that we don't know the cause, but what can you do.
